contracts/upgradeable_contracts/omnibridge_nft/modules/forwarding_rules/MultiTokenForwardingRulesConnector.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity 0.7.5;
+
+import "@openzeppelin/contracts/utils/Address.sol";
+import "./MultiTokenForwardingRulesManager.sol";
```

Comment:
Home similar the functionality of the modules `MultiTokenForwardingRules*` is similar to the same modules in the erc20 omnibridge contracts? Does it make sense to name the modules in the NFT OB with another prefix like `NFTForwardingRules*`?

Reply:
I didn't change them at all for now, so they are identical. I think we can rename them to avoid ambiguity.

Reply:
If they are identical, does it makes sense to move them in a separate package (like OZ provides) to import in both set of contracts? Maybe we can do this in the future to reduce the code duplication.

Reply:
Yes, I think it would make sense to do so in the future.